sensor_msgs/LaserScan laser_scan
sensor_msgs/Image rgb_image
sensor_msgs/Image depth_image
---
FieldComponent[] field_components
---
int8 components_detected